components/bottom-nav/Android/Android-bottom-nav.md

```diff
@@ -175,96 +159,177 @@ The following example shows a bottom navigation bar with four icons:
+| **Colored style** | `Widget.MaterialComponents.BottomNavigationView.Colored` | `?attr/colorPrimary` | `?attr/colorOnPrimary` at 60% | `?attr/colorOnPrimary` |
 
+Default style theme attribute: `?attr/bottomNavigationStyle`
 
-**Styles**
+**Note:** The `Widget.MaterialComponents.BottomNavigationView.PrimarySurface` style will automatically switch between between the component's primary colored style in light theme and surface colored style in dark theme. More information in the [Dark theme documentation](/third_party/java_src/android_libs/material_components/g3doc/theming/Dark.md#primary-vs-surface-coloring-for-large-surfaces).
 
-| &nbsp; | **Style** |
-| --- | --- |
-| **Default style** | |
-| **Icon style** | |
+See the full list of
+[styles](https://github.com/material-components/material-components-android/blob/master/lib/java/com/google/android/material/bottomnavigation/res/values/styles.xml)
+and
+[attributes](https://github.com/material-components/material-components-android/blob/master/lib/java/com/google/android/material/bottomnavigation/res/values/attrs.xml).
 
 
-## Theming a bottom nav bar
+## Theming a bottom navigation bar
 
-_Use the [Shrine theme](https://material.io/design/material-studies/shrine.html) for this example_
+Bottom navigation supports [Material Theming](https://material.io/components/bottom-navigation#theming) and can be customized in terms of color and typography.
 
+### Bottom navigation theming example
+
+API and source code:
+
+* `BottonNavigationView`
+    * [Class description](https://developer.android.com/reference/com/google/android/material/bottomnavigation/BottomNavigationView)
+    * [Class source](https://github.com/material-components/material-components-android/tree/master/lib/java/com/google/android/material/bottomnavigation/BottomNavigationView.java)
+
+The following example shows a bottom navigation bar with Material Theming.
+
+![Bottom navigation theming example, it has a pink background and four dark brown icons: favorites, music note, places, and news](assets/bottom-nav-theming.png)
+
+#### Implementing bottom navigation theming
+
+Using theme attributes and a style in `res/values/styles.xml` (themes all bottom navigation bars and affects other components):
+
+```xml
+<style name="Theme.App" parent="Theme.MaterialComponents.*">
+    ...
+    <item name="bottomNavigationStyle">@style/Widget.MaterialComponents.BottomNavigationView.Colored</item>
+    <item name="colorPrimary">@color/shrine_pink_100</item>
+    <item name="colorOnPrimary">@color/shrine_pink_900</item>
+</style>
 ```
-<code snippet here>
-The source code should render the following:
-* bottom navigation bar
-* four icons:
-    * favorites
-    * music note
-    * places (not part of material.io icons)
-    * news (not part of material.io icons)
+
+Or using a default style theme attribute, styles and a theme overlay (themes all bottom navigation bars but does not affect other components):
```

Comment:
I believe Google still uses the Oxford comma (thank goodness):

"Or using a default style theme attribute, styles, and a theme overlay..."